public/overview.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>FAF Voting App</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+          integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="icon" href="favicon.png"/>
+    <link rel="stylesheet" href="res/style/style.css" type="text/css"/>
+
+    <script src="res/scripts/httpget.js"></script>
+    <script>
+
+        /// On load, immediatly fetch voting subjects
+        window.onload = function (e) {
+            httpGet("/authed/getSubjects", receiveSubjects);
+        };
+
+        /// Parse all subjects
+        function receiveSubjects(json) {
+            let root = JSON.parse(json);
+            let subjects = root.data;
+            let receiver = document.getElementById('receiverZone');
+            receiver.innerHTML = '';
+
+            let pickSubjectH = document.createElement('h2');
+            pickSubjectH.textContent = "Votes overview";
+            receiver.appendChild(pickSubjectH);
+
+            if (subjects.length <= 0) {
+                pickSubjectH.textContent = "There are no votes";
+                return;
+            }
+
+            for (let i = 0; i < subjects.length; i++) {
+                let card = document.createElement("div");
+                card.className = "card";
+                receiver.append(card);
+
+                let cardHeader = document.createElement("div");
+                cardHeader.className = "card-header";
+                let subject = subjects[i];
+                cardHeader.innerHTML = subject.attributes.subject;
```

Comment:
Unless subject .attributes.subject contains HTML tags, you can use textContent - it's faster.

Reply:
no it does not

Reply:
Cool